src/shared/constants/endpoint.ts

```diff
@@ -0,0 +1 @@
+export const ENDPOINT = 'https://ll.thespacedevs.com/2.2.0';
```

Comment:
1.  Rename the file to `endpoints`, because this file is going to be extended.
2.  Move this file to `api` folder
3. And use enum `Endpoints`. For base endpoint can use `BASE_ENDPOINT`

Reply:
Done